### PR TITLE
website: Fix a couple typos in v0.82.0 release notes

### PR DESCRIPTION
**Pull request by einstein95**
# Description

Couple typos existed in the 0.82.0 release notes where a couple letters in the word were in the wrong order.

# Checklist

_Please tick the items as you have addressed them. Don't remove items; leave the ones that are not applicable unchecked._

I have:

- [x] followed the project's [contributing guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/dosbox-staging/dosbox-staging/blob/master/CODE_OF_CONDUCT.md).
- [x] performed a self-review of my code.
- [ ] commented on the particularly hard-to-understand areas of my code.
- [x] split my work into well-defined, bisectable commits, and I [named my commits well](https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#commit-messages).
- [x] applied the appropriate labels (bug, enhancement, refactoring, documentation, etc.)
- [ ] [checked](https://github.com/dosbox-staging/dosbox-staging/blob/main/scripts/compile_commits.sh) that all my commits can be built.
- [ ] confirmed that my code does not cause performance regressions (e.g., by running the Quake benchmark).
- [ ] added unit tests where applicable to prove the correctness of my code and to avoid future regressions.
- [ ] provided the release notes draft (for significant user-facing changes).
- [ ] made corresponding changes to the documentation or the website according to the [documentation guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md).
- [x] [locally verified](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md#previewing-documentation-changes-locally) my website or documentation changes.

### CODE REVIEW

File: website/docs/releases/release-notes/0.82.0.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## Summary
 
-Just six short months after our monstre [0.81.0 release](0.81.0.md), we're back with
+Just six short months after our monster [0.81.0 release](0.81.0.md), we're back with
```
Comment (johnnovak) [inline website/docs/releases/release-notes/0.82.0.md:7]: Nope, I wanted "monstre"  😎

File: website/docs/releases/release-notes/0.82.0.md [hunk 2]
```diff
@@ -682,7 +682,7 @@ For example:
   now also work.
 
 - The [JEMM memory manager](https://github.com/Baron-von-Riedesel/Jemm) now
-  works even with DOSBox Staging's VMware mouse emulated enabled (previuosly,
+  works even with DOSBox Staging's VMware mouse emulated enabled (previously,
```
Comment (johnnovak) [inline website/docs/releases/release-notes/0.82.0.md:685]: Legit typo 👍🏻

**Review by johnnovak — APPROVED** [review]
Thanks @einstein95

**Review by johnnovak — CHANGES_REQUESTED** [review]
Just remove the "monstre" part please 😄

**Comment by einstein95** [comment]
> Just remove the "monstre" part please 😄

Done, I've instead italicized the text as "monstre" isn't an English word (well, not any more)

**Comment by johnnovak** [comment]
> > Just remove the "monstre" part please 😄
> 
> Done, I've instead italicized the text as "monstre" isn't an English word (well, not any more)

Those online dictionaries are lying; English doesn't even have a central body of authority like German, so I'm not buying that 😄 

Anyway, I like the italics because it makes it look more dramatic 😆

**Review by johnnovak — APPROVED** [review]

**Pull request merged**